packages/firmware-release-config/jest.config.js

```diff
@@ -0,0 +1,5 @@
+const { ...baseConfig } = require('../../jest.config.base');
+
+module.exports = {
+    ...baseConfig,
+};
```

Comment:
nitcpick, when I am reading the PR by commits, this commit does not contain any test - only this config. The test is probably included in some other commit